spec/controllers/queue_items_controller_spec.rb

```diff
@@ -154,68 +138,42 @@
 
       it "should redirect to the my queue page" do
         set_current_user
-        post :update_queue, queue_items: [
-          { id: light_saber.id, position: 2 }, { id: r2d2.id, position: 1 }
-        ]
+        post :update_queue, queue_items: [{id: light_saber.id, position: 2}, {id: r2d2.id, position: 1}]
         expect(response).to redirect_to my_queue_path
       end
 
       it "reorders the queue items" do
-        post :update_queue, queue_items: [
-          { id: light_saber.id, position: 2 }, { id: r2d2.id, position: 1 }
-        ]
+        post :update_queue, queue_items: [{id: light_saber.id, position: 2}, {id: r2d2.id, position: 1}]
         expect(luke.queue_items).to eq([r2d2, light_saber])
       end
 
       it "normalizes the position numbers"  do
-        post :update_queue, queue_items: [
-          { id: light_saber.id, position: 3 }, { id: r2d2.id, position: 1 }
-        ]
+        post :update_queue, queue_items: [{id: light_saber.id, position: 3}, {id: r2d2.id, position: 1}]
```

Comment:
Line is too long. [104/80]<br>Space inside { missing.<br>Space inside } missing.